### PR TITLE
Use UTC time to maintain consistency

### PR DESCRIPTION
**Pull request by legokidlogan**

### CODE REVIEW

File: lua/autorun/server/sv_daily_restart.lua [hunk 1]
```diff
@@ -428,7 +428,7 @@ end
 local function getHoursUntilRestartHour()
     local hoursLeft = 24
     local restartHour = DESIRED_RESTART_HOUR
-    local currentHour = tonumber( os.date( "%H" ) )
+    local currentHour = tonumber( os.date( "!%H" ) )
```
Comment (plally) [inline lua/autorun/server/sv_daily_restart.lua:431]: Could also do `os.date( "!*t" ).hour`. Not much of a performance difference between the 2 though

File: lua/autorun/server/sv_daily_restart.lua [hunk 2]
```diff
@@ -445,8 +445,8 @@ end
 
 -- Calculates up to 23:59:59 to wait until restart
 local function waitUntilRestartHour()
-    local currentMinute = tonumber( os.date( "%M" ) )
-    local currentSecond = tonumber( os.date( "%S" ) )
+    local currentMinute = tonumber( os.date( "!%M" ) )
+    local currentSecond = tonumber( os.date( "!%S" ) )
```
Comment (plally) [inline lua/autorun/server/sv_daily_restart.lua:449]:
```suggestion
    local timeTbl = os.date( "!*t" )
    local currentMinute = timeTbl.min
    local currentSecond =  timeTbl.sec
```

Comment (legokidlogan) [reply lua/autorun/server/sv_daily_restart.lua:449]: `*t` causes it to generate all time formats, which would be less performant. I can combine the minute and seconds together in one string, however.

Comment (plally) [reply lua/autorun/server/sv_daily_restart.lua:449]: yeah but it doesnt have to do the string to int conversion

Comment (plally) [reply lua/autorun/server/sv_daily_restart.lua]: Please dont split it thats worse than both options

**Pull request merged**